### ISSUE TITLE
Shouldn't "max:4" be a check string length?

### ISSUE THREAD

**Issue by code4life49** [nylo-core/support#30]
![image](https://github.com/user-attachments/assets/5e6c8a77-547d-418f-b7a7-0154bd8edc86)

![image](https://github.com/user-attachments/assets/516d22bd-5a33-423c-912b-75c6482fd0aa)

hi, The logic of the code is not the same as the documentation says.

**Comment by agordn52** [comment]
Hey @code4life49,

This is working
``` dart
validate(rules: {
      "name": ["anthony", "max:8"],
    }, onSuccess: () {
      print("Validation passed");
    });
```

Also, this works
``` dart
validate(rules: {
      "name": [5, "max:8"],
    }, onSuccess: () {
      print("Validation passed");
    });
```

What value are you passing into the validator?

**Comment by code4life49** [comment]
@agordn52 hi，

```
validate(rules: {
      "password": ["12345678", "max:8"],
    }, onSuccess: () {
      print("Validation passed");
    });

```
I need to verify the password with a length of 8 digits, and if I enter "12345678", it will give an error, and if I enter "abcd1234", it will be correct. So, what I'm trying to make out is that max isn't good to use, and I don't know if the user is using a pure numeric string, or something else.

btw: My English is not very good, I hope I can express it okay.

**Comment by agordn52** [comment]
Thanks, I've fixed this is [5.82.2](https://github.com/nylo-core/support/releases/tag/v5.82.2).

In your Nylo project, you should now be able to update the framework using: `nylo_framework: ^5.32.8` 👍
